### DISCUSSION TITLE
Need help with install on Mac OS

### DISCUSSION THREAD

**Discussion by amddeus** [nextcloud/all-in-one#735] [Questions] [answered]
My steps: (new clean install)

1. install via curl -fsSL get.docker.com | sudo sh got an error
2. installed docker desktop via website (and changed disk image location to my external raid system)
3. used "sudo docker run -it \ --name nextcloud-aio-mastercontainer \ --restart always \ -p 80:80 \ -p 8080:8080 \ -p 8443:8443 \ --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \ --volume /var/run/docker.sock.raw:/var/run/docker.sock:ro \ nextcloud/all-in-one:latest"
4. opened ports 80, 443, 3478tcp and 3478udp on my router
5. Now I'm stuck at new AIO instance: Please type in the domain etc. (I have a DDNS with DYNU but I don't how to set it up with nextcloud docker)
6. i put in the ddns address and was hoping to then just use nextcloud locally  but I'm stuck at the containers status with apache and nextcloud not running
I've never used reverse proxies before and have no idea how to set them up
before I was using the Hanson nextcloud vm and was hoping to setup nc AIO the same network wise

**Comment by szaimen** [comment, marked as answer]
What is the error message when you type in your domain?

**Reply by amddeus** [reply]
No errors when in putting my domain. It ask me to start containers and when I do database, redis and talk start, but not apache and nextcloud

**Reply by szaimen** [reply]
Can you post the logs of the nextcloud container here please?
